identity-and-access-management/secret-manager/concepts.mdx

```diff
@@ -8,6 +8,38 @@ content:
 tags: secret-manager secret version
 ---
 
+
+<Concept>
+
+## Folder
+
+Folders are the underlying structure for organizing and grouping your secrets in one place. Folders are always within a [path](#path).
+
+</Concept>
+
+<Concept>
+
+## Path
+
+A path refers to the access path to your folders and secrets. Paths contain your folders, secrets and versions. Each path is prefixed with a slash.
+
+For example, if you create a folder `my-folder` within the path `/my-path`, your folder can be accessed at the `/my-path/my-folder` path.
+
+<Message type="important">
+  - A path can only contain folder names of 1 to 255 alphanumeric characters, dots, underscores and dashes.
```

Comment:
Unclear if it is 255 characters per folder name or 255 characters for the whole path